### PR TITLE
test: Improve the test case coverage of [sqlparser] module to 70%

### LINKED ISSUE #6508: task: Improve the test case coverage of [sqlparser] module to 70%
## Why you need it?
 Is your feature request related to a problem? Please describe in details

The test case coverage of [sqlparser] module is 44.68%, We need to go up to 70% to improve the robustness and stability of our code.
[sqlparser]模块的测试用例覆盖率为44.68%，我们需要提高到70%，以提高代码的健壮性和稳定性。

## How it could be?
A clear and concise description of what you want to happen. You can explain more about input of the feature, and output of it.

## Other related information

You can get some help from:
[https://seata.apache.org/docs/developers/contributor-guide/](https://seata.apache.org/docs/developers/contributor-guide/test-coverage-guide_dev)
https://seata.apache.org/blog/how-to-write-unit-tests

### PR DESCRIPTION
**Pull request by traitsisgiorgos**
- [x] I have registered the PR [changes](../changes).

### Ⅰ. Describe what this PR did
fix #6508

### CODE REVIEW

**Comment by traitsisgiorgos** [comment]
@slievrly @funky-eyes Hi, can you review my PR?

File: sqlparser/seata-sqlparser-core/src/test/java/org/apache/seata/sqlparser/EscapeSymbolTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.seata.sqlparser;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (xjlgod) [inline sqlparser/seata-sqlparser-core/src/test/java/org/apache/seata/sqlparser/EscapeSymbolTest.java]: Not use .* to import, maybe we can import what we really need.

Comment (traitsisgiorgos) [reply sqlparser/seata-sqlparser-core/src/test/java/org/apache/seata/sqlparser/EscapeSymbolTest.java]: done

File: changes/en-us/2.x.md [hunk 2]
```diff
@@ -161,6 +161,7 @@ Add changes here for all PR submitted to the 2.x branch.
 - [[#6375](https://github.com/apache/incubator-seata/pull/6375)] override console nested dependencies
 
 ### test:
+- [[#6608](https://github.com/apache/incubator-seata/pull/6608)] add unit test for sql-parser-core
```
Comment (lightClouds917) [inline changes/en-us/2.x.md:37]: please add it to zh-cn/2.x.md

File: sqlparser/seata-sqlparser-core/src/test/java/org/apache/seata/sqlparser/struct/ColumnMetaTest.java [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.seata.sqlparser.struct;
+
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (lightClouds917) [inline sqlparser/seata-sqlparser-core/src/test/java/org/apache/seata/sqlparser/struct/ColumnMetaTest.java]: do not import .*

File: sqlparser/seata-sqlparser-core/src/test/java/org/apache/seata/sqlparser/struct/IndexMetaTest.java [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.seata.sqlparser.struct;
+
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (lightClouds917) [inline sqlparser/seata-sqlparser-core/src/test/java/org/apache/seata/sqlparser/struct/IndexMetaTest.java]: do not import .*

File: sqlparser/seata-sqlparser-core/src/test/java/org/apache/seata/sqlparser/struct/TableMetaTest.java [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.seata.sqlparser.struct;
+
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (lightClouds917) [inline sqlparser/seata-sqlparser-core/src/test/java/org/apache/seata/sqlparser/struct/TableMetaTest.java]: do not import .*

**Review by lightClouds917 — CHANGES_REQUESTED** [review]

File: changes/zh-cn/2.x.md [hunk 6]
```diff
@@ -158,6 +158,7 @@
 - [[#6375](https://github.com/apache/incubator-seata/pull/6375)] 覆盖 console 前端安全漏洞
 
 ### test:
+- [[#6608](https://github.com/apache/incubator-seata/pull/6608)] add unit test for sql-parser-core
```
Comment (lightClouds917) [inline changes/zh-cn/2.x.md]: This is in Chinese. Thank you
添加sql-parser-core模块测试用例

**Review by lightClouds917 — CHANGES_REQUESTED** [review]

**Review by lightClouds917 — APPROVED** [review]
LGTM

**Comment by funky-eyes** [comment]
> LGTM

ok, let's run the CI workflow once to see the outcome.

File: changes/en-us/2.x.md [hunk 7]
```diff
@@ -22,6 +22,7 @@ Add changes here for all PR submitted to the 2.x branch.
 
 
 ### test:
+- [[#6608](https://github.com/apache/incubator-seata/pull/6608)] add unit test for sql-parser-core
```
Comment (funky-eyes) [inline changes/en-us/2.x.md:37]: Add your information to the list for `Thanks to these contributors`

File: changes/zh-cn/2.x.md [hunk 8]
```diff
@@ -20,6 +20,8 @@
 ### security:
 
 ### test:
+- [[#6608](https://github.com/apache/incubator-seata/pull/6608)] 添加sql-parser-core模块测试用例
```
Comment (funky-eyes) [inline changes/zh-cn/2.x.md:37]: Add your information to the list for Thanks to these contributors

**Review by funky-eyes — COMMENTED** [review]
` Coverage     51.13%   49.52%   -1.61%   `
Unit test coverage did not meet expectations.

File: changes/en-us/2.x.md [hunk 9]
```diff
@@ -42,7 +43,9 @@ Thanks to these contributors for their code commits. Please report an unintended
...
 - [funky-eyes](https://github.com/funky-eyes)
 - [tanyaofei](https://github.com/tanyaofei)
+- [traitsisgiorgos](https://github.com/traitsisgiorgos)
 - [wanghongzhou](https://github.com/wanghongzhou)
 - [ggbocoder](https://github.com/ggbocoder)
 
-Also, we receive many valuable issues, questions and advices from our community. Thanks for you all.
\ No newline at end of file
+
+Also, we receive many valuable issues, questions and advices from our community. Thanks for you all.
```
Comment (funky-eyes) [inline changes/en-us/2.x.md]:
```suggestion
Also, we receive many valuable issues, questions and advices from our community. Thanks for you all.
```

**Comment by xjlgod** [comment]
Hi, I use your code to verify test coverage in my local idea env. Its test coverage is not high enough, can you add more unit tests? 
image

**Review by lightClouds917 — APPROVED** [review]
our goal is to improve test coverage to 70%,can you add more? thanks for your job.

**Review by funky-eyes — APPROVED** [review]
LGTM
I will merge this PR and look forward to you continuing to increase the unit test coverage for this module

**Pull request merged**